docs/current/guides/snippets/use-secrets/external/main.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import anyio
+import dagger
+from google.cloud import secretmanager
+
+async def main():
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
+
+        # get secret from Google Cloud Secret Manager
+        secretPlaintext = await gcp_get_secret_plaintext("PROJECT-ID", "SECRET-ID")
```

Comment:
Following the comment on this function, remove the `await`:

```suggestion
        secretPlaintext = gcp_get_secret_plaintext("PROJECT-ID", "SECRET-ID")
```